packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -42,10 +42,43 @@ export default class FunctionRun extends Command {
       path: flags.path,
       userProvidedConfigName: flags.config,
       callback: async (_app, ourFunction) => {
+        let functionExport = '_start'
+
+        if (flags.export !== undefined) {
+          this.log('Using export ${flags.export} from the --export flag.')
+          functionExport = flags.export
+        } else if (
+          ourFunction.configuration.targeting !== undefined &&
+          ourFunction.configuration.targeting.length > 0
+        ) {
+          const targeting = ourFunction.configuration.targeting
+
+          if (targeting.length > 1 && isTTY()) {
+            let targets = targeting.map((target) => ({
+              label: target.target,
+              value: target.export || '_start',
+            }))
+
+            functionExport = await renderAutocompletePrompt({
+              message: `Which target would you like to execute?`,
+              choices: targets,
+            })
+          } else if (targeting.length === 1) {
```

Comment:
You know here that it's greater than 0 and not greater than 1 so I think `=== 1` is the only possibility for this code branch